website/pages/docs/drivers/java.mdx

```diff
@@ -97,6 +97,15 @@ task "web" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| Exec | false |
+| FSIsolation | FSIsolationNone  |
+| NetIsolationModes | NetIsolationModeHost, NetIsolationmodeGroup |
+| MountConfigs | MountConfigSupportNone |
```

Comment:
@tgross are these true on Linux?  I think that the macOS version of the Java driver tosses FS Isolation, but I thought the Linux version is like exec+java